adal/src/telemetry/java/com/microsoft/aad/adal/CacheEvent.java

```diff
@@ -86,6 +86,10 @@ public void processEvent(final Map<String, String> dispatchMap) {
         dispatchMap.put(EventStrings.TOKEN_TYPE_IS_MRRT, "");
         dispatchMap.put(EventStrings.TOKEN_TYPE_IS_RT, "");
 
+        if (dispatchMap.containsKey(EventStrings.SPE_INFO)) {
```

Comment:
@GTValentine Changes to make `CacheEvent` consistent with `HttpEvent` as discussed